e2e/tests/test_cases/node_disconnection_test.robot

```diff
@@ -0,0 +1,95 @@
+*** Settings ***
+Documentation    Node disconnection test
+...              https://longhorn.github.io/longhorn-tests/manual/pre-release/node-not-ready/node-disconnection/node-disconnection/
+
+Test Tags    manual_test_case
+
+Resource    ../keywords/common.resource
+Resource    ../keywords/volume.resource
+Resource    ../keywords/setting.resource
+Resource    ../keywords/network.resource
+Resource    ../keywords/statefulset.resource
+Resource    ../keywords/workload.resource
+
+Test Setup    Set test environment
+Test Teardown    Cleanup test resources
+
+*** Variables ***
+${LOOP_COUNT}    3
+${RETRY_COUNT}    300
+${RETRY_INTERVAL}    1
+
+*** Test Cases ***
+Node Disconnect And Keep Data Writing And No Replica On The Disconnected Node
+    Given Set setting auto-salvage to false
+
+    When Create volume 0 with 10 GB and 3 replicas
+    And Attach volume 0 to node 1
+    And Wait for volume 0 healthy
+    And Update volume 0 replica count to 2
+    And Delete volume 0 replica on volume node
```

Comment:
If we always have the need to create a volume with no replica on the attached node, can we make it a keyword instead?

Reply:
Updated